test_wishlists.py

```diff
@@ -19,6 +19,43 @@ def setUp(self):
 	def tearDown(self):
 		wishlists.db=DatabaseEngine()
 
+	def test_wishlists(self):
+		resp = self.app.get('/wishlists')
+		self.assertEqual(resp.status_code, status.HTTP_200_OK)
+		self.assertTrue(len(resp.data) > 0)
+
+	def test_read_wishlist(self):
+		resp = self.app.get('/wishlists/1')
+		self.assertEqual(resp.status_code, status.HTTP_200_OK)
+		data = json.loads(resp.data)
+		self.assertEqual(data['user_id'], 'user1')
+
+
+	def test_read_wishlist_not_found(self):
+		resp = self.app.get('/wishlists/2')
+		self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
+
+	def test_item(self):
+		resp = self.app.get('/wishlists/1/items')
+		self.assertEqual(resp.status_code, status.HTTP_200_OK)
+		self.assertTrue(len(resp.data) > 0)
+
+	def test_item_not_found(self):
+		resp = self.app.get('/wishlists/2/items')
+		self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
+	def test_read_wishlist_item(self):
+		resp = self.app.get('/wishlists/1/items/item1')
+		self.assertEqual(resp.status_code, status.HTTP_200_OK)
+		data = json.loads(resp.data)
+		self.assertEqual(data['description'], 'test item 1')
+
+	def test_read_wishlist_item_item_not_found(self):
```

Comment:
No, it is not the same, different wishlist, one returns error because the wishlist not exist, the other is no such item in an existed wishlist.